osu.Game.Rulesets.Mania/Edit/ManiaSelectionHandler.cs

```diff
@@ -31,13 +29,16 @@ private void load(IAdjustableClock clock)
             editorClock = clock;
         }
 
-        public override void HandleDrag(SelectionBlueprint blueprint, DragEvent dragEvent)
+        public override void HandleMovement(MoveSelectionEvent moveEvent)
         {
-            adjustOrigins((ManiaSelectionBlueprint)blueprint);
-            performDragMovement(dragEvent);
-            performColumnMovement(dragEvent);
+            var maniaBlueprint = (ManiaSelectionBlueprint)moveEvent.Blueprint;
+            int lastColumn = maniaBlueprint.HitObject.HitObject.Column;
```

Comment:
The nested `HitObject`s had me in a double take, but I guess there's not really a better way to name them...

Reply:
what about renaming the DHO `DrawbleObject`?

Reply:
It's something we can consider and discuss, but not necessary for now. It would need to be done everywhere (e.g. `Playfield.AllHitObjects`, `HitObjectContainer.Objects`, `HitObjectContainer.AliveObjects`.

Out of scope for this PR.